src/test/java/org/abstractj/kalium/crypto/StreamTest.java

```diff
@@ -0,0 +1,33 @@
+package org.abstractj.kalium.crypto;
+
+import org.abstractj.kalium.NaCl;
+import org.junit.BeforeClass;
+import org.junit.Test;
+

```

Comment:
Please give me rights to modify StreamTest class, I refactored it using constants I added to TetsVectors. See pull request #32